src/SerialFirmata.h

```diff
@@ -23,6 +23,10 @@
 #include <SoftwareSerial.h>
 #endif
 
+//uncomment SERIAL_STORE_AND_FORWARD to collect bytes received by serial port until the receive buffer 
+// gets filled or a data gap is detected to avoid forwarding single bytes at baud rates below 50000
+//#define SERIAL_STORE_AND_FORWARD
```

Comment:
what do you think about renaming `SERIAL_STORE_AND_FORWARD` to `FIRMATA_SERIAL_BUFFER_INPUT`?